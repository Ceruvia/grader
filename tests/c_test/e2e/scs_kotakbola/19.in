847 526 790
H 27
H 64
B 70
H 23
M 12
M 62
M 86
B 57
H 22
H 43
B 29
M 83
B 51
M 89
B 54
H 15
B 72
B 93
B 4
H 44
M 87
M 58
B 8
H 50
M 52
E
