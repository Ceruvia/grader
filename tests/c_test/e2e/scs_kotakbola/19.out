288 529 438
Tidak
